package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-firebase-crash",
-  "version": "4.1.2",
+  "version": "4.2.0",
```

Please rollback version change from PR. I'll publish new release with tag as soon as the PR will me merged.
```suggestion
  "version": "4.1.2",
```

---

Done